Control coverage/README.md

```diff
@@ -59,3 +59,56 @@ Following controls in AzTS are currently externally scanned:
+| Azure_RedisCache_DP_Use_SSL_Port | Non-SSL port must not be enabled for Redis Cache | Non-SSL port must not be enabled for Redis Cache | Redis Cache should allow access only via SSL |
+| Azure_ServiceFabric_DP_Set_Property_ClusterProtectionLevel | The ClusterProtectionLevel property must be set to EncryptAndSign for Service Fabric clusters |The ClusterProtectionLevel property must be set to EncryptAndSign for Service Fabric clusters | Service Fabric clusters should have the ClusterProtectionLevel property set to EncryptAndSign |
+| Azure_SQLDatabase_AuthZ_Use_AAD_Admin | Use AAD Authentication for SQL Database | Enable Azure AD admin for the SQL Database | SQL servers should have an Azure Active Directory administrator provisioned |
+| Azure_SQLDatabase_DP_Enable_TDE | Enable Transparent Data Encryption on SQL databases | Enable Transparent Data Encryption on SQL databases | Transparent Data Encryption on SQL databases should be enabled |
+
+## Frequently Asked Questions (FAQ)
+
+<br>
+
+**Even after remediating my resource, it is still showing as failed against controls in AzTS UI. The controls depends on MDC Assessment. What should I do?**
+
+**NOTE:** *Kindly make sure that the resource(s) is(are) already fixed. The controls which depends on MDC assessment could be found [here](#list-of-controls-that-depends-on-microsoft-defender-for-cloud-mdc-in-azure-tenant-security-azts).* 
+
+1. Go to **Azure Portal**.
+2. Search for **Microsoft Defender for Cloud** and **open** that.
+
+    ![Image](../Images/MDCEvaluationImage1.png)
+
+3. Click on **Recommendation under the General tab**, in the left side panel.
+
+    ![Image](../Images/MDCEvaluationImage2.png)
+
+4. Click on the **All Recommendations**.
+5. Search for the related [recommendations](#list-of-controls-that-depends-on-microsoft-defender-for-cloud-mdc-in-azure-tenant-security-azts) and open it.
+
+    ![Image](../Images/MDCEvaluationImage3.png)
+
+6. Check the list of **unhealthy resources** to see if your resource is present in that list or not.
+7. If your resource(s) is not present in unhealthy resources list, run the scan from AzTS UI and check the status of your resource(s).
+7. If your resource(s) is present in **unhealthy resources list** and **'Fix' button is available** in the bottom, select the resource(s) that you need to remediate and click on 'Fix' button and wait till your resource(s) show up in **healthy resources list**.
+
+    ![Image](../Images/MDCEvaluationImage4.png)
+ 
+8. If your resource(s) is present in **unhealthy resources list** and **'Fix' button is not available** in the bottom, you have to wait till the MDC evaluation is refreshed and wait till your resource(s) show up in **healthy resources list**. You can find the **refresh interval** at the top. 
```

Comment:
Can we add one screenshot for this as well?

Reply:
Will add one.